src/basic/rlimit-util.c

```diff
@@ -30,6 +30,12 @@
 #include "string-table.h"
 #include "time-util.h"
 
+#define MAX_NICE 20
+
+static inline int rlimit_to_nice(int prio) {
```

Comment:
rlimits use the "rlim_t" type, please use that here too for the rlimit value

Reply:
done.